workflow/controller/controller.go

```diff
@@ -834,7 +834,7 @@ func (wfc *WorkflowController) processNextItem(ctx context.Context) bool {
 
 	woc := newWorkflowOperationCtx(wf, wfc)
 
-	if !wfc.throttler.Admit(key.(string)) {
+	if !(woc.GetShutdownStrategy().Enabled() && woc.GetShutdownStrategy() == wfv1.ShutdownStrategyTerminate) && !wfc.throttler.Admit(key.(string)) {
```

Comment:
@tczhao Quick suggestion:
Can we handle this state workflow in the informer filter section so, the workflow will not even get into the throttler queue?
https://github.com/argoproj/argo-workflows/blob/59d7e5b129d736ae1f34c51eaccb96dd3cb0d7be/workflow/controller/controller.go#L992

Reply:
The informer filter only exclude the workflow.
We want to include the shutdown workflow in the wfqueue, and process it.
Here we skip throttle/parallelism check if the workflow is marked shutdown

Reply:
oh yeah got it thanks.  I thought opposite way. make sense